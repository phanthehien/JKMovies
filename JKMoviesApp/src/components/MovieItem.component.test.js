/**
 * Created by hien.phanthe on 3/16/17.
 */

import React from 'react';
import MovieItem from './MovieItem.component'

import renderer from 'react-test-renderer';

test('render Contact Form correctly', () => {

    var movie = {
        title:"Flight of Fury",
        type: "Horror",
        director: "Xuan Nguyen",
        length: 123,
        short_description:"This is a cool film",
        photo_url: "http://movie.com/photo/fligh.png"
    };

    const component = renderer.create(
        <MovieItem movie = {movie} />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot()
});
