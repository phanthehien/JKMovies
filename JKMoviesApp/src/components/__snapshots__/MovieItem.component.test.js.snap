// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Contact Form correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "green",
      "borderWidth": 1,
      "height": 100,
      "justifyContent": "center",
      "margin": 0,
      "marginTop": -1,
      "padding": 15,
      "paddingBottom": 30,
      "paddingTop": 30,
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "http://movie.com/photo/fligh.png",
      }
    }
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "blue",
        "fontWeight": "bold",
      }
    }
  >
    Flight of Fury
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Horror
  </Text>
</View>
`;
